src/main/resources/html/script.js

```diff
@@ -349,8 +473,8 @@ Visualizer.prototype.makeMap = function(){
     var svgMap = $target.svg("get");
 
     // Fill it with circles
-    for(var i in this.towns){
-        var town = this.towns[i];
+    for(let i in this.towns){
```

Comment:
Wow. I think this is like the first time I've seen the JavaScript `let` in the wild. I feel like I'm living in the future. :wink: